parseSite/translator.js

```diff
@@ -0,0 +1,76 @@
+const readline      = require('readline');
+const request       = require('request');
+const queryString   = require('query-string');
+const fs            = require('fs');
+
+if(!fs.existsSync('./apiKey.txt')){
+    return console.log('Файл с ключём для API (apiKey.txt) не найден!');
+}
+
+const apiKey = fs.readFileSync('./apiKey.txt', 'utf8');
+
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+function enterText() {
+    return new Promise((resolve, reject) => {
+        rl.question('Наберите текст для перевода с end на rus: ', (text) => {
+            if(text === "q")
+                reject(text);
+            resolve(text);
+        });
+    });
+}
+
+function translite(userText) {
+    return new Promise((resolve, reject) => {
+        const stringified = queryString.stringify({
+            key:apiKey,
+            text:userText,
+            lang:'en-ru',
+        });
+
+        request.post('https://translate.yandex.net/api/v1.5/tr.json/translate?'+stringified, {},(err, response, body) => {
```

Comment:
Тогда не придется парсить json ручками
```suggestion
        request.post({url: 'https://translate.yandex.net/api/v1.5/tr.json/translate?'+stringified, json: true}, {},(err, response, body) => {
```